modules/text2vec-openai/clients/vectorizer_test.go

```diff
@@ -28,6 +28,52 @@ import (
 	"github.com/weaviate/weaviate/modules/text2vec-openai/ent"
 )
 
+func TestBuildUrlFn(t *testing.T) {
+	t.Run("buildUrlFn returns default OpenAI Client", func(t *testing.T) {
+		config := ent.VectorizationConfig{
+			Type:         "",
+			Model:        "",
+			ModelVersion: "",
+			ResourceName: "",
+			DeploymentID: "",
+			BaseURL:      "https://api.openai.com/v1",
```

Comment:
Can we agree on unifying `baseURL` value in all OpenAI modules to be `https://api.openai.com` so that we would introduce this setting in a unified manner across all OpenAI modules